var/spack/repos/builtin/packages/pocl/package.py

```diff
@@ -91,8 +91,7 @@ def cmake_args(self):
 
     @run_after('install')
     def symlink_opencl(self):
-        with working_dir(self.build_directory):
-            os.symlink("OpenCL", join_path(self.prefix.include, "CL"))
+        os.symlink("CL", os.path.join(self.prefix.include, "OpenCL"))
```

Comment:
`self.prefix.include.OpenCL` should work with `join()`.

Reply:
Not sure if my second commit addresses your concern, but certainly using the attribute magic of the Prefix object makes the code clearer.